.run/createReportForTaskSearch.run.xml

```diff
@@ -0,0 +1,16 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="createReportForTaskSearch" type="WdioConfigurationType">
+    <node-interpreter>project</node-interpreter>
+    <node-options />
+    <wdio-package>$PROJECT_DIR$/client/node_modules/@wdio/cli</wdio-package>
+    <working-directory>$PROJECT_DIR$/client</working-directory>
+    <pass-parent-env>true</pass-parent-env>
+    <envs>
+      <env name="SERVER_URL" value="http://localhost:8180" />
+    </envs>
+    <wdio-config-file-path>$PROJECT_DIR$/client/config/wdio.config.js</wdio-config-file-path>
+    <wdio-framework>Mocha</wdio-framework>
+    <test-file>$PROJECT_DIR$/client/tests/webdriver/baseSpecs/createReportForTaskSearch.spec.js</test-file>
+    <method v="2" />
+  </configuration>
+</component>
```

Comment:
Please put this file in a separate branch/PR with other run configurations, it is not directly related to this change.